recipes/server_redhat.rb

```diff
@@ -53,7 +53,7 @@
 end
 
 # If using PGDG, add symlinks so that downstream commands all work
-if node['postgresql']['enable_pgdg_yum'] == true
+if node['postgresql']['enable_pgdg_yum'] == true || node['postgresql']['use_pgdg_packages'] == true
   [
```

Comment:
We still want symlinks created for necessary commands so they'll exist in the $PATH after the cookbook runs, even if we don't set ```node['postgresql']['enable_pgdg_yum']``` to true.